blog/docs/releases/announcing-knative-v1-4-release.md

```diff
@@ -0,0 +1,209 @@
+---
+title: "v1.4 release"
+linkTitle: "v1.4 release"
+Author: "[Nader Ziada](https://twitter.com/nziada)"
+Author handle: https://github.com/nader-ziada
+date: 2022-04-20
+description: "Knative v1.4 release announcement"
+type: "blog"
+---
+
+
+## Announcing Knative 1.4 Release
+
+A new version of Knative is now available across multiple components.
+
+Follow the instructions in
+[Installing Knative](https://knative.dev/docs/install/) to install the components you require.
+
+## Table of Contents
+- [Highlights](#highlights)
+- [Serving v1.4](#serving-v1.4-NOSPACES)
```

Comment:
These links are broken, something to do with `NOSPACES`?
Check out the 1.3 release markdown https://raw.githubusercontent.com/knative/docs/main/blog/docs/releases/announcing-knative-v1-3-release.md

Reply:
fixed links